xchange-core/src/main/java/org/knowm/xchange/service/trade/params/CancelOrderByInstrument.java

```diff
@@ -0,0 +1,8 @@
+package org.knowm.xchange.service.trade.params;
+
+import org.knowm.xchange.instrument.Instrument;
+
+public interface CancelOrderByInstrument extends CancelOrderParams {
```

Comment:
to refactor towards using `Instrument` every where I think we should also make changes like this in the `*ByCurrencyPair` interfaces:

```
/**
  * @use CancelOrderByInstrument 
  */
@Deprecated
public interface CancelOrderByCurrencyPair extends CancelOrderByInstrument {

  default Instrument getInstrument() {
    return getCurrencyPair();
  }

  CurrencyPair getCurrencyPair();
}
```